src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveGatherVectorIndices.template

```diff
@@ -470,17 +470,7 @@ namespace JIT.HardwareIntrinsics.Arm
 
         private void ValidateResult({Op1BaseType}[] firstOp, {Op2BaseType}[] secondOp, {Op3BaseType}[] thirdOp, {RetBaseType}[] result, [CallerMemberName] string method = "")
         {
-            bool succeeded = true;
-
-            for (var i = 0; i < RetElementCount; i++)
-            {
-                {RetBaseType} gatherResult = ({RetBaseType})(firstOp[i] == 0 ? 0 : ({ExtendedElementType})secondOp[thirdOp[i]]);
-                if (result[i] != gatherResult)
-                {
-                    succeeded = false;
-                    break;
-                }
-            }
+            var succeeded = Helpers.CheckGatherVectorBehavior<{RetBaseType}, {ExtendedElementType}, {Op3BaseType}>(firstOp, secondOp, thirdOp, result);
```

Comment:
was there anything we were missing for this test, or is the change just trying to reuse the new logic that is written in `Helpers.cs`?

Reply:
Re-using the logic. If we can extract out how we validate/check the APIs in the helpers, we should.